README.md

```diff
@@ -38,14 +38,14 @@ end_insert -->
 
 <!-- cargo-sync-readme start -->
 
-Rusqlite Migration is a simple and performant schema migration library for [rusqlite](https://lib.rs/crates/rusqlite).
+Rusqlite Migration is a simple and performant schema migration library for [rusqlite](https://crates.io/crates/rusqlite).
 
 * **Performance**:
     * *Fast database opening*: to keep track of the current migration state, most tools create one or more tables in the database. These tables require parsing by SQLite and are queried with SQL statements. This library uses the [`user_version`][uv] value instead. It’s much lighter as it is just an integer at a [fixed offset][uv_offset] in the SQLite file.
     * *Fast compilation*: this crate is very small and does not use macros to define the migrations.
 * **Simplicity**: this crate strives for simplicity. Just define a set of SQL statements as strings in your Rust code. Add more SQL statements over time as needed. No external CLI required. Additionally, rusqlite_migration works especially well with other small libraries complementing rusqlite, like [serde_rusqlite][].
 
-[diesel_migrations]: https://lib.rs/crates/diesel_migrations
+[diesel_migrations]: https://crates.io/crates/diesel_migrations
```

Comment:
I like lib.rs but it’s better to point to an official source that will be around in 10 years, hopefully.